scripts/sv/fermiLite/scanBam.sh

```diff
@@ -36,5 +36,5 @@ KMER_KILL_LIST=$(echo "${REF_FASTA}" | sed 's/.fasta$/.kill.kmers/')
     --cluster "${CLUSTER_NAME}" \
     --num-executors 20 \
     --driver-memory 30G \
-    --executor-memory 30G \
+    --executor-memory 40G \
```

Comment:
I believe that if you go over 32Gb you get 8-byte references rather than 4.  We should fight hard to stay below 32Gb, unless I'm wrong.